docs/uma-tokenholders/guidence-on-collateral-currency-addition.md

```diff
@@ -0,0 +1,35 @@
+### Guidance for the Addition of Collateral Currencies
+
+Collateral currencies can be added through the collateral currency UMIP template available on Github, by adding 
+ - the name of the token to the be added 
+ - the contract address of the token  together with a link to the address on etherscan
+ - the amount that should be added to the store as final fee expressed in units of the intended collateral currency.   This should equate to approximately $400 at the time the UMIP is submitted.
+
+***Note**: rebasing tokens cannot be added as collateral currencies as they do not work with UMA's factory contracts*
+
+____________
+The below provides some guidelines when using collateral currencies in an UMA contract
+
+**Stablecoins**
+Consider carefully whether to treat stablecoins as their intended peg, or as the current market value.
+
+**Non-traded tokens**
+Some tokens, such as interest bearing tokens or liquidity tokens may have no direct market.  
+
+If using such tokens in a contract which requires collateral monitoring, such as the EMP, that there is a clear way for liquidity to be obtained should it be required.  It may be useful to add this to the UMIP, under the security section.
+
+**Volitility**
+Where a proposed collateral type has high volitility, a higher store fee than the recommended $400 may be appropriate.
```

Comment:
Is this something we should articulate a bit more? As a writer, I would have questions like:
- How much higher should I make the store fee?
- Do I HAVE to or is it just a suggestion?

Reply:
volatility.

What would be a "higher` store fee? We should provide recommendation, such as $1000

Reply:
Well, my idea was just to leave it open to their own interpretation of how volatile their token is and how much more they should add

Reply:
Its a suggestion.